src/test/store/profile-view.test.js

```diff
@@ -1067,13 +1067,13 @@ describe('actions/ProfileView', function() {
         getState()
       );
       const keys = [...screenshotMarkersById.keys()];
-      expect(keys.length).toEqual(1);
+      expect(keys.length).toEqual(2);
 
       const [screenshots] = [...screenshotMarkersById.values()];
       if (!screenshots) {
         throw new Error('No screenshots found.');
       }
-      expect(screenshots.length).toEqual(profile.threads[0].markers.length);
+      expect(screenshots.length).toEqual(profile.threads[0].markers.length / 2);
```

Comment:
Maybe add a comment above this line, like this:
```js
// The profile contains the same markers twice, but with different window ids.
// `screenshots` contains the screenshots for only one window id.
// That's why we need to halve the markers length when comparing the 2 lengths.
```
to explain the `/2`.

Reply:
I added the comment!

Thanks!